From 01dd31da1f5f32dc2676bb0ce13434259b89cdad Mon Sep 17 00:00:00 2001
From: Nicolas Boulenguez <nicolas@debian.org>
Date: Mon, 28 Dec 2020 23:42:39 +0100
Subject: [PATCH 04/21] Compute D_CROSS from P_CROSS

If someone overrides the package name, they will expect the
installation directory to change.
---
 debian/rules | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/rules b/debian/rules
index 3b4f1de..bd7b1eb 100755
--- a/debian/rules
+++ b/debian/rules
@@ -245,7 +245,7 @@ _gnu_type	= $(shell dpkg-architecture -a$(1) -qDEB_HOST_GNU_TYPE -f 2>/dev/null)
 _multiarch	= $(shell dpkg-architecture -a$(1) -qDEB_HOST_MULTIARCH -f 2>/dev/null)
 CROSS_GNU_TYPE	= $(subst i586,i686,$(call _gnu_type,$(CROSS_ARCH)))
 P_CROSS		= binutils-$(subst _,-,$(CROSS_GNU_TYPE))
-D_CROSS		= debian/binutils-$(subst _,-,$(CROSS_GNU_TYPE))
+D_CROSS		= debian/$(P_CROSS)
 
 # If $(TARGET) is not set, try reading debian/target
 ifeq (,$(TARGET))
-- 
2.30.1

